.envrc

```diff
@@ -1,3 +1,3 @@
-export SUI_NETWORK="testnet"
+export SUI_NETWORK="localnet"
```

Comment:
@eivindingebrigtsen I'm not sure why this was set to `testnet` - surely we want to test against the local sui node??